packages/core/src/scp-cf/destination.ts

```diff
@@ -126,11 +130,11 @@ interface DestinationJSON {
 interface DestinationConfiguration {
   URL: string;
   Name?: string;
-  ProxyType: string;
+  ProxyType?: string;
```

Comment:
I just wonder if making something less strict is a breaking change in general. It will not actively break something, but could change the behavior of the code. Is there a strict definition of a breaking change?

Reply:
@marikaner I see your point now, makes sense.
@FrankEssenberger making something less optional should be a compatible change, since you're relaxing requirements (assuming your code is aligned with your type definitions. if your code fails with `Cannot read property 'ProxyType'  of undefined`, that's bad).
The way I look at it, a breaking change is a change that breaks existing code (hence the name). Any that makes code that worked at one point in time break at a later point in time is breaking. So this does not only include API changed but also changes in the state space (which we fortunately do not have to worry much about in the SDK). But if, we for example, refactor the destination accessor to not aggressively cache destinations any more, that can be a breaking change.